### PR TITLE
feat: added radio and checkbox button

### PR DESCRIPTION
**Pull request by tanishq20**
- Added radio button
- Added checkbox button
- Styled Input fields with validation

Live preview link: https://deploy-preview-27--compact-ui.netlify.app/docs/components/input

preview image

### CODE REVIEW

File: docs/components/input.html [hunk 1]
```diff
@@ -102,42 +103,108 @@ <h3 class="main-subhead">Input fields</h3>
...
+                        style="width: 100%; height: 411px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                    </iframe>
```
Comment (shuntzhere) [inline docs/components/input.html]: You might want to delete one closing tag.

File: stylesheets/components/input.css [hunk 2]
```diff
@@ -29,20 +29,53 @@
...
 .required-mark {
   font-size: 1.6rem;
   color: var(--danger-600);
 }
+
+.input-selector {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 1rem;
```
Comment (shuntzhere) [inline stylesheets/components/input.css:72]: Utility classes can be created for these.

**Review by shuntzhere — APPROVED** [review]
Code is clean, like the use of classes. Also, suggested some minor changes.

File: stylesheets/components/input.css (hunk 2 above)
Comment (tanishq20) [reply stylesheets/components/input.css:72]: Yeah, will add in dev and then implement it.

**Review by coldpigli — APPROVED** [review]
Loved the intuitiveness of the classnames used. Abstract and well formatted.
Go ahead and merge

File: docs/components/input.html (hunk 1 above)
Comment (tanishq20) [reply docs/components/input.html]: Done

**Pull request merged**